news/_posts/translation/2020-09-09-do-you-speak-prestashop-august-2020.md

```diff
@@ -32,7 +32,7 @@ I would like to thank gcocozza, jmcoget and jbonello who offered to open and hel
 
 The PrestaShop Day for Italy will take place next 24th September as an online event!
 35 live sessions in Italian are planned with entrepreneurs, partners, tech experts and more. Don’t forget to mark this event on your calendar! :calendar:
-To discover more about this event and to register, I invite you to check this link: https://bit.ly/2CSQlc2
+To discover more about this event and to register, I invite you to [check this link](https://bit.ly/2CSQlc2)
```

Comment:
what about:
```suggestion
To discover more about this event and to register, I invite you to visit [PSD Italy site web](https://bit.ly/2CSQlc2)
```

Reply:
You're right. I have just edited the document. Thank you